3.3/.sti/bin/assemble

```diff
@@ -8,12 +8,14 @@ set -e
 echo "---> Copying application source ..."
 cp -Rf /tmp/src/. ./
 
-if [ -f setup.py ]; then
-    echo "---> Installing application ..."
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    echo "---> Installing dependencies ..."
-    pip install --user -r requirements.txt
+if [[ -f requirements.txt ]]; then
+  echo "---> Installing dependencies ..."
+  pip install --user -r requirements.txt
+fi
+
+if [[ -f setup.py ]]; then
+  echo "---> Installing application ..."
+  python setup.py develop --user
```

Comment:
@mfojtik @soltysh I changed this from `install` to `develop` as mentioned in my previous comment.